databases/data_models/migrations/20210527092816_initial-schema.up.sql

```diff
@@ -4,36 +4,26 @@ CREATE TABLE IF NOT EXISTS dwh_content_units_measures
     event_unit_uid               text COLLATE "default" NOT NULL,
     content_unit_type_name       character varying COLLATE "default",
     all_events_count             bigint,
-    unique_users_curday_count    bigint,
+    unique_users_last10min_count bigint,
```

Comment:
Is this now really 10min_count?

Are more PR expected soon?

Reply:
it will be, working on it now :)